### PR TITLE
charts: disable validating webhook configuration

### PR DESCRIPTION
**Pull request by shashankram**
**Description**:
Since the validating webhook has been disabled by c48acb9c due to
issue #2264, the validating webhook config needs to be disabled as
well. Otherwise config-map update will be broken.

**Affected area**:

- New Functionality      [ ]
- Documentation          [ ]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [X]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
`No`

### CODE REVIEW

File: charts/osm/values.yaml [hunk 1]
```diff
@@ -83,3 +83,7 @@ OpenServiceMesh:
     # Destination's API or collector endpoint where the spans will
     # be sent to
     endpoint: "/api/v2/spans"
+
+  # Set to true to enable the osm-config validator webhook
+  # Currently disabled due to Issue #2264
+  enableConfigValidator: false
```
Comment (nojnhuh) [inline charts/osm/values.yaml:89]: Any reason to create a new value here vs. just commenting out the config temporarily to mirror c48acb9c61c8650a7ff0b4516de001c3d1728387? I don't see why anyone would ever set this to `true` while the issue gets fixed.

Comment (shashankram) [reply charts/osm/values.yaml:89]: Is there a Helm equivalent of `#if 0` ?

Comment (nojnhuh) [reply charts/osm/values.yaml:89]: You can use `{{/* a comment */}}` or I believe `{{ if false }}` would work too.

Comment (shashankram) [reply charts/osm/values.yaml:89]: Perfect, will try this.

**Review by nojnhuh — APPROVED** [review]

**Review by ksubrmnn — APPROVED** [review]

**Review by draychev — APPROVED** [review]

**Pull request merged**